<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserHandlogDialog</class>
 <widget class="QDialog" name="UserHandlogDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employees Management</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="10,3,1">
     <property name="spacing">
      <number>24</number>
     </property>
     <item>
      <widget class="QTableView" name="tw_usersInfo"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,2,5,2,1">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbtnAddEmployee">
         <property name="text">
          <string>Add Employee</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtnDelEmp">
         <property name="text">
          <string>Del  Employee</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbtn_close">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbtn_close</sender>
   <signal>clicked()</signal>
   <receiver>UserHandlogDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>448</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
